airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2238,10 +2469,16 @@ components:
           $ref: "#/components/schemas/ConnectionSchedule"
         status:
           $ref: "#/components/schemas/ConnectionStatus"
+        operationIds:
```

Comment:
why do we have both operations and operation ids in the same struct? should operations just contain their own id in the operations struct??

Reply:
This is part of `WbConnectionRead` where you also have:
- `sourceId` and `source` which is a `SourceRead` that contains a `sourceId`
- `destinationId` and `destination` which is a `DestinationRead` that contains a `destinationId`

I guessed that `WbConnectionRead` is an object aimed to ease queries for the front-end to the API/backend? (compared to using a `ConnectionRead` where you would need to do a second API call to retrieve the details of each source/destination/operation too)